docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,437 @@
+	const name = get(path('users', 'current', 'name'));
+
+	return <h1>{`Hello, ${name}`}</h1>;
+});
+```
+
+This middleware contains an `executor` method that can be used to run processes on the store.
+
+```tsx
+import { create } from '@dojo/framework/core/vdom';
+import store from '../middleware/store';
+import logout from '../processes/logout';
+import { State } from '../../interfaces';
+
+const factory = create({ store }).properties();
+export const User = factory(function User({ middleware: { store } }) {
+	const { get, path } = store;
+	const name = get(path('users', 'current', 'name'));
+
+	const onLogOut = () => {
+		store.executor(logout)({});
+	};
+
+	return (
+		<h1>
+			{`Hello, ${name}`}
+			<button onclick={onLogOut}>Log Out</button>
+		</h1>
+	);
+});
+```
+
+### StoreProvider
+
+A `StoreProvider` is a Dojo widget that has its own `renderer` and connects to the store. It is always encapsulated in another widget because it does not defined its own properties.
+
+> widget/User.ts
+
+```tsx
+import { create } from '@dojo/framework/core/vdom';
+import { State } from '../../interfaces';
+
+const factory = create().properties();
+export const User = factory(function User() {
+	return (
+		<StoreProvider
+			stateKey="state"
+			paths={(path) => [path('users', 'current')]}
+			renderer={(store) => {
+				const { get, path } = store;
+				const name = get(path('users', 'current', 'name'));
+
+				return <h1>{`Hello, ${name}`}</h1>;
+			}}
+		/>
+	);
+});
+```
+
+The `StoreProvider` occurs as part of `User`'s render and provides its own `renderer` like any other Dojo widget.
+
+### Container
+
+A `Container` is a widget that fully encapsulates another widget. It connects the store to the widget with a `getProperties` function.
+
+> widget/User.tsx
+
+```tsx
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+interface UserProperties {
+	name?: string;
+}
+const factory = create().properties<UserProperties>();
+export const User = factory(function User({ properties }) {
+	const { name = 'Stranger' } = properties();
+	return <h1>{`Hello, ${name}`}</h1>;
+});
+```
+
+> widget/User.container.ts
+
+```ts
+import { createStoreContainer } from '@dojo/framework/stores/StoreContainer';
+import { State } from '../interfaces';
+import User from './User';
+
+const StoreContainer = createStoreContainer<State>();
+
+const UserContainer = StoreContainer(User, 'state', {
+	getProperties({ get, path }) {
+		const name = get(path('user', 'current', 'name'));
+
+		return { name };
+	}
+});
+```
+
+In this example `UserContainer` wraps `User` to display the current user's name. `createStoreContainer` is a wrapper that gets used to ensure the proper typing of `getProperties`. `getProperties` is responsible for accessing data from the store and creating properties for the widget.
+
+A `StoreContainer`'s properties are a 1:1 mapping to the widget it wraps. The widget become the properties of the `StoreContainer`, but they are all optional.
```

Comment:
"The widget become the properties" I am assuming this means something like `The widget is passed the properties` or `The widget receives the properties`? If this isn't a mistake maybe some more clarification is in order?